help/en/releasenotes/current-draft-note.shtml

```diff
@@ -612,6 +612,10 @@
                 <a href="https://www.jmri.org/help/en/html/doc/Technical/Patterns.shtml#typescript">Typescript and Javascript support</a>
                 has been updated.</li>
             <li>The included 'ajv' and 'fast-deep-equal' web packages have been updated.</li>
-	    <li>Improved XSLT display of many elements in panel XML files.</li>
+            <li>Improved XSLT display of many elements in panel XML files.</li>
+            <li>Remove the <em>Store Only table content (No Panels)...</em> option from the File menu
+            used by the PanelPro tables.</li>
+            <li>Don't load panels (PE/CPE/LE/SE) during xml data file loading when running the
```

Comment:
I suggest that this line is changed to:
`JMRI doesn't load panels (PE/CPE/LE/SE) during xml data file loading when running the`

When I read `Don't load panels ...`, I got the impression that I, as a JMRI user, shouldn't load panels during xml data file loading.

Reply:
See if the updated text is better.

Reply:
Works fine. Thanks!